src/test/java/com/at/globalclasses/ApiTools.java

```diff
@@ -5,12 +5,17 @@
 import java.io.InputStream;
 import java.util.Properties;
 
+import com.mongodb.client.FindIterable;
```

Comment:
Why do you add this dependencies?

Reply:
I will delete it. I add this because I thought that i went to use it.